File: DesafiosPython/URI1046.py
data = input().split(' ')
dinit = int(data[0])
dfinal = int(data[1])

if dinit < dfinal:
    horas = dfinal - dinit
else:
    horas = (24 - dinit) + dfinal
print(f'O JOGO DUROU {horas} HORA(S)')

File: DesafiosJava/URI1046.java
package DesafiosJava;

import java.util.Scanner;

public class URI1046 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        String[] valores = entrada.nextLine().split(" ");
        entrada.close();

        int horaInicial = Integer.parseInt(valores[0]);
        int horaFim = Integer.parseInt(valores[1]);
        int duracaoJogo;
        
        if (horaFim <= horaInicial){
            duracaoJogo = (24 - horaInicial) + horaFim;
            
        }
        else{
            duracaoJogo = horaFim - horaInicial;
        }  
        System.out.println("O JOGO DUROU " + duracaoJogo + " HORA(S)");
    }
}
